api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Suite.java

```diff
@@ -99,6 +102,20 @@ public Suite(String correlationId, String company, String project, String name,
     this.patternCorrelationIds = patternCorrelationIds;
   }
 
+  Suite(String correlationId, String company, String project, String name,
```

Comment:
I'm not sure if providing new constructor just like this is a good practice. I would rather prefer to have one constructor.